src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,49 +1,78 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
 
-export const TodoModal: React.FC = () => {
-  return (
-    <div className="modal is-active" data-cy="modal">
-      <div className="modal-background" />
-
-      {true ? (
-        <Loader />
-      ) : (
-        <div className="modal-card">
-          <header className="modal-card-head">
-            <div
-              className="modal-card-title has-text-weight-medium"
-              data-cy="modal-header"
-            >
-              Todo #2
-            </div>
+interface Props {
+  selectedTodo: Todo
+  isTodoSelected: boolean,
+  setIsTodoSelected: (value: boolean) => void,
+}
+
+export const TodoModal: React.FC<Props> = ({
+  selectedTodo,
+  isTodoSelected,
+  setIsTodoSelected,
+}) => {
+  const [user, setUser] = useState<User>();
+
+  useEffect(() => {
+    getUser(selectedTodo.userId)
+      .then(setUser)
+      // eslint-disable-next-line no-console
+      .catch((error) => console.log(error));
+  }, []);
+
+  if (isTodoSelected) {
+    return (
+      <div className="modal is-active" data-cy="modal">
+        <div className="modal-background" />
+
+        {!user ? (
```

Comment:
Add a boolean `useState` which will show if it loading or not. 
```suggestion
        {isLoading? (
```